gemhardware/include/gem/hw/amc13/AMC13Manager.h

```diff
@@ -119,6 +120,8 @@ namespace gem {
             xdata::Boolean monBackPressure;
             xdata::Boolean enableLocalTTC;
 
+	    xdata::Boolean enableLEMO;
```

Comment:
Add this parameter into the localTriggerConfig bag (and the corresponding parameter to the L1AInfo class)